CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.5.1
+
+* Documentation improvements
+
```

Comment:
I hope you don't mind releasing a new version to bring the doc on docs.us updated